### PR TITLE
update cchq_prbac_grandfather_privs.py

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

File: corehq/apps/hqadmin/management/commands/cchq_prbac_grandfather_privs.py [hunk 1]
```diff
@@ -37,7 +37,8 @@ class Command(BaseCommand):
         )
     )
 
-    def handle(self, *privs, **kwargs):
+    def handle(self, priv, **kwargs):
```
Comment (snopoke) [inline corehq/apps/hqadmin/management/commands/cchq_prbac_grandfather_privs.py]: Shouldn't the options also be converted?

Comment (snopoke) [inline corehq/apps/hqadmin/management/commands/cchq_prbac_grandfather_privs.py]: And this might need to support multiple pirvilege args

**Comment by nickpell** [comment]
✅

**Comment by nickpell** [comment]
@emord cc: @sravfeyn

File: corehq/apps/hqadmin/management/commands/cchq_prbac_grandfather_privs.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/hqadmin/management/commands/cchq_prbac_grandfather_privs.py]: updated

**Review by emord — APPROVED** [review]
Is it more standard to have `handle` before `add_arguments` or vice versa?

**Comment by nickpell** [comment]
Most of the time I've been putting ```add_arguments``` before ```handle``` (although apparently not this time).  Don't know if there's any convention around this, but I like putting the inputs to the command before the implementation because it follows the spirit of writing code like a [newspaper article](https://medium.com/@kentcdodds/newspaper-code-structure-8b4616af9539).

**Pull request merged**